examples/py/reddit/demo.ipynb

```diff
@@ -58,11 +153,14 @@
    "metadata": {},
    "outputs": [
     {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "[1, 2]\n",
-      "123\n"
+     "ename": "AttributeError",
+     "evalue": "'builtins.Graph' object has no attribute 'add_vertex_properties'",
+     "output_type": "error",
+     "traceback": [
+      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
+      "\u001b[0;31mAttributeError\u001b[0m                            Traceback (most recent call last)",
+      "Cell \u001b[0;32mIn[3], line 8\u001b[0m\n\u001b[1;32m      5\u001b[0m g\u001b[39m.\u001b[39madd_vertex(timestamp\u001b[39m=\u001b[39m\u001b[39m2\u001b[39m, \u001b[39mid\u001b[39m\u001b[39m=\u001b[39m\u001b[39m\"\u001b[39m\u001b[39mben\u001b[39m\u001b[39m\"\u001b[39m, properties\u001b[39m=\u001b[39m{\u001b[39m\"\u001b[39m\u001b[39mproperty 1\u001b[39m\u001b[39m\"\u001b[39m: \u001b[39m2\u001b[39m, \u001b[39m\"\u001b[39m\u001b[39mproperty 2\u001b[39m\u001b[39m\"\u001b[39m: \u001b[39m0.6\u001b[39m, \u001b[39m\"\u001b[39m\u001b[39mproperty 4\u001b[39m\u001b[39m\"\u001b[39m: \u001b[39mFalse\u001b[39;00m})\n\u001b[1;32m      6\u001b[0m g\u001b[39m.\u001b[39madd_vertex(timestamp\u001b[39m=\u001b[39m\u001b[39m3\u001b[39m, \u001b[39mid\u001b[39m\u001b[39m=\u001b[39m\u001b[39m\"\u001b[39m\u001b[39mben\u001b[39m\u001b[39m\"\u001b[39m, properties\u001b[39m=\u001b[39m{\u001b[39m\"\u001b[39m\u001b[39mproperty 2\u001b[39m\u001b[39m\"\u001b[39m: \u001b[39m0.9\u001b[39m, \u001b[39m\"\u001b[39m\u001b[39mproperty 3\u001b[39m\u001b[39m\"\u001b[39m: \u001b[39m\"\u001b[39m\u001b[39mhello\u001b[39m\u001b[39m\"\u001b[39m, \u001b[39m\"\u001b[39m\u001b[39mproperty 4\u001b[39m\u001b[39m\"\u001b[39m: \u001b[39mTrue\u001b[39;00m})\n\u001b[0;32m----> 8\u001b[0m g\u001b[39m.\u001b[39;49madd_vertex_properties(\u001b[39mid\u001b[39m\u001b[39m=\u001b[39m\u001b[39m\"\u001b[39m\u001b[39mben\u001b[39m\u001b[39m\"\u001b[39m, properties\u001b[39m=\u001b[39m{\u001b[39m\"\u001b[39m\u001b[39munchanging property\u001b[39m\u001b[39m\"\u001b[39m: \u001b[39m123\u001b[39m})\n\u001b[1;32m     10\u001b[0m \u001b[39mprint\u001b[39m(g\u001b[39m.\u001b[39mvertex(\u001b[39m\"\u001b[39m\u001b[39mben\u001b[39m\u001b[39m\"\u001b[39m)\u001b[39m.\u001b[39mproperties\u001b[39m.\u001b[39mtemporal\u001b[39m.\u001b[39mget(\u001b[39m\"\u001b[39m\u001b[39mproperty 1\u001b[39m\u001b[39m\"\u001b[39m)\u001b[39m.\u001b[39mvalues())\n\u001b[1;32m     11\u001b[0m \u001b[39mprint\u001b[39m(g\u001b[39m.\u001b[39mvertex(\u001b[39m\"\u001b[39m\u001b[39mben\u001b[39m\u001b[39m\"\u001b[39m)\u001b[39m.\u001b[39mproperties\u001b[39m.\u001b[39mconstant\u001b[39m.\u001b[39mget(\u001b[39m\"\u001b[39m\u001b[39munchanging property\u001b[39m\u001b[39m\"\u001b[39m))\n",
```

Comment:
This add_vertex_properties should now be `add_constant_properties()` in the vertex itself - hence the error you are getting